### PR TITLE
Change model Ourtem to OurTeam

### PR DESCRIPTION
**Pull request by JiwonHwang**
voila

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
LGTM!

File: content/migrations/0006_merge.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```
Comment (jinpark) [inline content/migrations/0006_merge.py:1]: I think this makes it so theres two 0006 migrations. Seems like it'll break migrations?

**Comment by jinpark** [comment]
@RachellCalhoun did you test this before merging?

**Comment by RachellCalhoun** [comment]
no....I haven't done tests yet. this weekend.

**Comment by jinpark** [comment]
I meant did you run this to see if it worked. I have a suspicion that this might break migrations

**Comment by RachellCalhoun** [comment]
nope, I thought we had set up automated testing? wasn't that the point of tests?

**Comment by jinpark** [comment]
How can we have automated testing without any tests? :D

And, I dont think @hassanabidpk setup automated testing yet.

Also, automated testing doesnt catch migration errors.

**Comment by jinpark** [comment]
@RachellCalhoun can you run this code locally and see if it works for you and reply back here if it works :)

https://help.github.com/articles/syncing-a-fork/

**Comment by jinpark** [comment]
Ran this locally and came back with

```
(lightandleadership) ➜  lightandleadership git:(master) python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/contrib/admin/apps.py", line 22, in ready
    self.module.autodiscover()
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/contrib/admin/__init__.py", line 24, in autodiscover
    autodiscover_modules('admin', register_to=site)
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/site-packages/django/utils/module_loading.py", line 74, in autodiscover_modules
    import_module('%s.%s' % (app_config.name, module_to_search))
  File "/Users/jinpark/.pyenv/versions/lightandleadership/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/Users/jinpark/Documents/code/lightandleadership/content/admin.py", line 2, in <module>
    from .models import OurStory, Ourteam, EduProgram
ImportError: cannot import name 'Ourteam'
```

Looks like theres more work to be done @JiwonHwang

**Pull request merged**